### DISCUSSION TITLE
How can I use "render" function to render CLI text without overrding previous one?

### DISCUSSION THREAD

**Discussion by tal-rofe** [vadimdemedes/ink#548] [Q&A]
There is a scenario where I want to render 2 components lets say. But I don't want to use `render` once, but use `render` and then after another `render`. But if I do so - the 2nd `render` will override the output of the first one. Is there any way to overcome this issue?

**Comment by tal-rofe** [comment]
Currently, best solution as I see is to use this code (suppose your `stdout` is `process.stdout`):
```ts
render(...);
process.stdout.write('WHATEVER');
```

**Comment by vadimdemedes** [comment]
Not sure what the use case is, so it's hard to understand how to help without more details and example code.
